sumpy/expansion/local.py

```diff
@@ -47,6 +43,31 @@ class LocalExpansionBase(ExpansionBase):
 """
 
 
+class LocalExpansionBase(ExpansionBase):
+    """Base class for local expansions.
+    """
+    def m2l_global_precompute_exprs(self, src_expansion, src_rscale,
+            dvec, tgt_rscale, sac):
+        """Return an iterable of expressions that needs to be precomputed
+        for multipole-to-local translations that depend only on the
+        distance between the multipole center and the local center.
+
+        Since there are only a finite amount of different values for the
+        distance between per level, these can be precomputed for the tree.
```

Comment:
```suggestion
        Since there are only a finite number of different values for the
        distance between per level, these can be precomputed for the tree.
```